quizapp.js

```diff
@@ -0,0 +1,131 @@
+const quizQuestion = [
+  {
+    id: 1,
+    question: "What is the capital of Denmark?",
+    options: [
+      { text: "Berlin", isCorrect: false },
+      { text: "Copenhagen", isCorrect: true },
+      { text: "Madrid", isCorrect: false },
+      { text: "Rome", isCorrect: false },
+    ],
+    explanation: "Copenhagen is the capital of Denmark.",
+  },
+  {
+    id: 2,
+    question: "What is the capital of India?",
+    options: [
+      { text: "Delhi", isCorrect: true },
+      { text: "Tamil Nadu", isCorrect: false },
+      { text: "hyderabad", isCorrect: false },
+      { text: "Mumbai", isCorrect: false },
+    ],
+    explanation: "Delhi is the capital of Denmark.",
+  },
+  {
+    id: 3,
+    question: "What is the currency used in Denmark?",
+    options: [
+      { text: "Kroner", isCorrect: true },
+      { text: "Pound", isCorrect: false },
+      { text: "Dollar", isCorrect: false },
+      { text: "Rupees", isCorrect: false },
+    ],
+    explanation: "Kroner is the currency of Denmark.",
+  },
+  {
+    id: 4,
+    question:
+      "The quick brown fox jumps over the lazy dog near the riverbank, where the sunlight glistens on the water and the gentle breeze rustles through the tall grass and trees?",
+    options: [
+      { text: "Kroner", isCorrect: true },
+      { text: "Pound", isCorrect: false },
+      { text: "Dollar", isCorrect: false },
+      { text: "Rupees", isCorrect: false },
+    ],
+    explanation: "Kroner is the currency of Denmark.",
+  },
+];
+//creating form and styling the form
+const form = document.createElement("form");
+document.body.appendChild(form);
+form.style.alignItems = "center";
+form.style.margin = "2%";
+
+//creating div element for question and answers
+for (let i = 0; i < quizQuestion.length; i++) {
+  if (quizQuestion[i].question.length < 140) {
+    const divQuestion = document.createElement("div");
+    divQuestion.setAttribute("id", "question - " + quizQuestion[i].id);
+    form.appendChild(divQuestion);
+    divQuestion.innerText =
+      quizQuestion[i].id + " . " + quizQuestion[i].question;
+
+    const divAnswers = document.createElement("div");
+    divAnswers.setAttribute("id", "answers");
+    divQuestion.appendChild(divAnswers);
+
+    for (let j = 0; j < quizQuestion[i].options.length; j++) {
+      const button = document.createElement("button");
+      button.setAttribute("class", "btnoptions");
+      button.innerText = quizQuestion[i].options[j].text;
+      button.isCorrect = quizQuestion[i].options[j].isCorrect;
+
+      divAnswers.appendChild(button);
+      button.style.display = "block";
```

Comment:
I would define this in the `style.css` as you want all buttons to have this property

Reply:
I have changed the code based on the requirements and implemented all your feedback. styling given in separate CSS file and wrapping the code in the DOMContentLoaded function and wrapping  logic into separate functions.I have given the condition separately for validating the question field. Thank you for your feedback.